src/main/java/seedu/address/ui/ProfilePanel.java

```diff
@@ -22,6 +23,8 @@
 public class ProfilePanel extends UiPart<Region> {
 
     public static final String DEFAULT_MESSAGE = "Ain't Nobody here but us chickens!";
+    public static final String PERSON_DOES_NOT_EXIST_MESSAGE = "This person does not exist in Lisa.\n"
+            + "Would you like to add him/her?";
```

Comment:
phrasing this as a question means you are asking input from the user, might not wanna do that

Reply:
Agreed!

Reply:
I deleted it for now, but maybe we can consider saying "You can use the add command to add him/her" ? Giving a bit of suggestion can be nice and warm!